Library/Formula/libantlr3c.rb

```diff
@@ -14,8 +16,30 @@ class Libantlr3c < Formula
   end
 
   def install
-    system "./configure", "--disable-dependency-tracking",
-                          "--prefix=#{prefix}"
-    system "make install"
+    args = ["--disable-dependency-tracking",
+            "--enable-64bit",
```

Comment:
cvc4 will segfault if it's not built with `--enable-64bit`. Should the cvc4 formula fail if `MacOS.prefer_64_bit?` is false?